### PR TITLE
Add support for PAYMETHOD parameter

### PR DESCRIPTION
**Pull request by pavol-tuka**
Allow to set different payment method (https://www.gpwebpay.cz/downloads/GP_webpay_HTTP_EN.pdf):
```
CRD – payment card
GPAY – GooglePay
APAY – Apple Pay
BTNCS – Platba 24
SOFORT
EPS
PAYSAFECARD
SEPADIRECTDEBIT
KLARNA
```

### CODE REVIEW

**Comment by newPOPE** [comment]
Hello,

what about creating Enum with paymethod-s for easiest access to correct values via constants?

**Comment by pavol-tuka** [comment]
It would be nice to have feature. More robust solution would create `PaymentRequestInterface` which allow developer to implement custom PaymentRequest which can be used to set all parameters, not only PAYMETHOD.

**Comment by pavol-tuka** [comment]
Constants added.

File: src/AdamStipak/Webpay/PaymentRequest.php [hunk 1]
```diff
@@ -3,6 +3,7 @@
 namespace AdamStipak\Webpay;
 
 use AdamStipak\Webpay\PaymentRequest\AddInfo;
+use AdamStipak\Webpay\PaymentRequest\PaymentMethod;
```
Comment (newPOPE) [inline src/AdamStipak/Webpay/PaymentRequest.php]: Not sure that this class exists, if not pls delete it.

**Review by newPOPE — CHANGES_REQUESTED** [review]

**Pull request merged**